testing/trino-tests/src/test/java/io/trino/execution/TestCoordinatorDynamicFiltering.java

```diff
@@ -397,7 +367,14 @@ private void assertQueryDynamicFilters(@Language("SQL") String query, TupleDomai
 
     private void assertQueryDynamicFilters(Session session, @Language("SQL") String query, TupleDomain<ColumnHandle> expectedTupleDomain)
     {
-        expectedDynamicFilter = expectedTupleDomain;
+        expectedCoordinatorDynamicFilter = expectedTupleDomain;
+        if (!isEnableCoordinatorDynamicFiltersDistribution(session) && getJoinDistributionType(session).equals(PARTITIONED)) {
+            expectedTableScanDynamicFilter = TupleDomain.all();
```

Comment:
we can expect `expectedTableScanDynamicFilter` for broadcast join too

Reply:
we do, we're skipping it only when coordinator DF distribution is off and the join type is PARTITIONED